content/spin/dynamic-configuration.md

```diff
@@ -166,3 +167,30 @@ url = "redis://localhost"
 ```
 
 You must individually grant each component access to the stores that it needs to use. To do this, use the `component.key_value_stores` entry in the component manifest within `spin.toml`. See [Spin Key Value Store](kv-store-api-guide.md) for more details. 
+
+### SQLite Storage Runtime Configuration
+
+Spin provides built-in SQLite storage. By default, this is backed by a database that Spin creates for you. However, you can use the Spin runtime configuration file (`runtime-config.toml`) to add and customize local SQLite databases.
+
+The following example `runtime-config.toml` tells Spin to map the `default` database to a SQLite database elsewhere in the file system:
+
+```toml
+[sqlite_database.default]
+path = "/planning/todo.db"
+```
+
+If you need more than one database, you can configure multiple stores, each with their own name:
```

Comment:
```suggestion
If you need more than one database, you can configure multiple stores, each with its name:
```

Reply:
"each with its own name" is fine; it makes enough sense. 
Initially, I was coming from the point that the "own" name is redundant because an entity's name is that entity's own name by default. However, I do apologise if we start to split hairs; not my intention. Sometimes things just jump out, and I am compelled to tweak.

Reply:
Well, I might possibly be able to identify with that...!